### PR TITLE
Add documentation about functions to IR.md

### PR DESCRIPTION
**Pull request by gramalingam**
Signed-off-by: Ganesan Ramalingam <grama@microsoft.com>

Add documentation about functions to IR.md

### CODE REVIEW

File: docs/IR.md [hunk 1]
```diff
@@ -171,6 +171,41 @@ The ‘status’ property indicates whether the syntax, semantics, or presence o
 
 There are two distinct ways to pass information to operators – inputs and attributes. Inputs represent graph inputs or values computed elsewhere in the graph, while attributes are used for values that are constants in the graph. This distinction may be highly relevant to achieving good performance for some implementations, while completely irrelevant to others.
 
+### Functions
+
+A _function_ may be thought of as an operator combined with an implementation of the operator using
+other, more primitive, ops, referrred to as the _function body_. The function body consists of a
```
Comment (liqunfu) [inline docs/IR.md:177]: referrred -> referred

File: docs/IR.md [hunk 2]
```diff
@@ -171,6 +171,41 @@ The ‘status’ property indicates whether the syntax, semantics, or presence o
...
+output|string[]|The output parameters of the function.
+node|Node[]|A list of nodes, forming a partially ordered computation graph. It must be in topological order.
+|opset_import|OperatorSetId|A collection of operator set identifiers used by the function implementation.
+
+The name and domain serve to identify the operator uniquely. An opset version is not explicitly
+identified in a FunctionProto, but it is implicitly determined by the opset version of the domain
+included in the model.
+
+The input, output, and attribute constitute the signature part of the operator. No type information
+is explicitly incuded in the signature.
```
Comment (liqunfu) [inline docs/IR.md:205]: incuded -> included

File: docs/IR.md (hunk 1 above)
Comment (gramalingam) [reply docs/IR.md]: Fixed, thanks!

File: docs/IR.md (hunk 2 above)
Comment (gramalingam) [reply docs/IR.md]: Fixed, thanks!

**Review by liqunfu — APPROVED** [review]

**Pull request merged**